test/index.js

```diff
@@ -1,95 +1,130 @@
-var tape = require('tape');

```

Comment:
I’m using Sublime Text EditorConfig plugin; at first I was thinking the problem is on my side, but then I looked in `.editorconfig` and it says `indent_style = tab` so I guess originals weren’t correct since they were spaces :/